pyserini/search/_searcher.py

```diff
@@ -48,6 +50,14 @@ def __init__(self, index_dir: str):
         self.object = JSimpleSearcher(JString(index_dir))
         self.num_docs = self.object.getTotalNumDocuments()
 
+    @classmethod
+    def from_prebuilt_index(cls, prebuilt_index_name_or_path: str):
```

Comment:
why `_path` here? "ms-marco-passage", stored in `INDEX_MAPPING`,  is just a name right?

Reply:
It's just a name. Users can also provide an index directory to build a searcher. I can remove `_path` if you want to use prebuilt_index only.

Reply:
yea, this method should be "name" only - following hgf, right? With hgf, you initialize with an identifier, not an explicit path...?

Reply:
hgf `from_pretrained` method also supports a path to a directory.

from_pretrained(pretrained_model_name_or_path, *model_args, **kwargs):
pretrained_model_name_or_path –
either:
+ shortname/identifier
+ a path to a directory containing model weights saved using save_pretrained(), e.g.: ./my_model_directory/


I'm a bit curious here, do we want this function to follow hgf convention or support name only?

Reply:
hrm. didn't realize that about hgf. 

Let's just go with "shortname/identifier".

Because the path is simply our normal constructor:

```python
searcher = SimpleSearcher('indexes/index-robust04-20191213/')
```